src/app/inventory/DraggableInventoryItem.tsx

```diff
@@ -110,7 +114,14 @@ function DraggableInventoryItem({
   const [touchActive, setTouchActive] = useState(false);
   const longPressed = useRef<boolean>(false);
   const timer = useRef<number>(0);
-  const isNotTransferable = item.notransfer && (!item.isEngram || !item.canPullFromPostmaster);
+  const stores = useSelector(sortedStoresSelector);
```

Comment:
This makes sense, but we shouldn't add redux subscriptions to this component - it'd slow down DIM. We may have to find another way to do this that doesn't require looking at stores.